src/app/campaign/store/models/campaign.models.ts

```diff
@@ -32,3 +32,10 @@ export const docToCampaign = (doc: HalDoc): Campaign => {
   campaign.set_account_uri = doc.expand('prx:account');
   return campaign;
 };
+
+export const duplicateCampaign = (campaign: Campaign): Campaign => {
+  // remove id from dup campaign and set to Draft
+  const { id, ...dupCampaign } = campaign;
```

Comment:
Maybe more of a UX / @brandonhundt thing ... but I would prefer if the duplicated flights/campaigns added `"DUPLICATE #{prev_title}"` to the title or something.  Otherwise a bit hard to tell what happened - this thing works so fast, and the page hardly changes, so it's hard to tell what just happened when I click duplicate.

Reply:
This is good thinking. Duplicated Flights (within the campaign) do get the suffix of " (Copy)" appended to their name. And/or maybe it should have a confirmation, "Are you sure you want to duplicate?" Or a toast notification that the duplication has been successful. It almost shouldnt be a disappearing toast though because I feel like it should contain instruction to set the flight dates. I don't feel like those interactions haven't been thought about too much in this application.

Reply:
Just my thoughts!  I'm fine with merging this and figuring it out later too.

Reply:
Honestly, yeah I think we'll have a few afterthoughts like that on this project. I haven't been feeling like things are fully flushed out UXwise.